src/Integration/MefServices/IActiveSolutionBoundTracker.cs

```diff
@@ -18,5 +18,11 @@ public interface IActiveSolutionBoundTracker : IDisposable
         /// Returns whether the active solution is bound to a SonarQube project
         /// </summary>
         bool IsActiveSolutionBound { get; }
+
+        /// <summary>
+        /// Event to notify subscribers when the analyzers of a solution might have changed.
+        /// This occurs when a new solution is opened, or the SonarQube binding status of the solution changes.
+        /// </summary>
+        event EventHandler AnalyzersChanged;
```

Comment:
The `solutionTracker.ActiveSolutionChanged` doesn't trigger if the binding changes. It triggers when you open/close a solution. However we need a way to force a full Roslyn reanalysis, which is needed when the binding status changes.